media-tv/tvbrowser-bin/tvbrowser-bin-4.2.1.ebuild

```diff
@@ -27,11 +27,10 @@ src_install() {
 
 	# Generate launcher
 	exeinto /opt/bin
-	newexe - tvbrowser <<-_EOF_
-		#!/bin/bash
-		export JAVA_OPTS="\${JAVA_OPTS} -Dpropertiesfile=/opt/${P}/linux.properties"
-		exec /bin/bash /opt/${P}/tvbrowser.sh "\$@"
-	_EOF_
+	sed -e "s/^P=.*\$/P=${P}/" \
```

Comment:
I think ||die is missing. Or make it a patch

Reply:
added || die